extra/mariabackup/xtrabackup.cc

```diff
@@ -1088,7 +1088,8 @@ struct my_option xb_client_options[]= {
      (G_PTR *) &xtrabackup_print_param, (G_PTR *) &xtrabackup_print_param, 0,
      GET_BOOL, NO_ARG, 0, 0, 0, 0, 0, 0},
     {"use-memory", OPT_XTRA_USE_MEMORY,
-     "The value is used instead of buffer_pool_size",
+     "The value is used in place of innodb_buffer_pool_size. " 
+	 "This option is only relevant when the --prepare option is specified.",
```

Comment:
Not sure why this indentation shows up in git diff but not in my IDE

Reply:
the line began with a tab. I've corrected this the latest push. Also previous line ended in a space.

Thanks for the fix.